DataFormats/FWLite/interface/EventSetup.h

```diff
@@ -105,7 +106,7 @@ namespace fwlite
       edm::EventID m_syncedEvent;
       edm::Timestamp m_syncedTimestamp;
       
-      TFile* m_file;
+      edm::propagate_const<TFile*> m_file;
```

Comment:
The two functions are meant to be logically const. Unfortunately ROOT doesn't give const interfaces to allow the work to happen. So let's just mark `m_file` as mutable.